### PR TITLE
feat: add cmd to generate ipns webfinger document

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/orb-cli/ipnswebfingergencmd/webfingergen.go [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+				return fmt.Errorf("failed to send http request: %w", err)
+			}
+
+			webFingerDoc := strings.ReplaceAll(string(respBytes), "$publicKeyHashValue", keyID)
+
+			if errMkdir := os.MkdirAll(fmt.Sprintf("%s/website/.wellknow", webFingerDir), os.ModePerm); errMkdir != nil {
+				return errMkdir
+			}
+
+			webFingerFile := fmt.Sprintf("%s/website/.wellknow/webfinger", webFingerDir)
```
Comment (sandrask) [inline cmd/orb-cli/ipnswebfingergencmd/webfingergen.go:155]: typo? Should it be ./well-known or ./wellknown?

**Review by sandrask — APPROVED** [review]

File: cmd/orb-cli/ipnswebfingergencmd/webfingergen.go (hunk 1 above)
Comment (fqutishat) [reply cmd/orb-cli/ipnswebfingergencmd/webfingergen.go]: done

File: pkg/discovery/endpoint/restapi/operations.go [hunk 2]
```diff
@@ -186,6 +193,24 @@ func (o *Operation) webFingerHandler(rw http.ResponseWriter, r *http.Request) {
 		writeResponse(rw, resp, http.StatusOK)
 	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, o.webCASPath)):
 		o.handleWebCASQuery(rw, resource)
+	case resource == o.baseURL:
+		resp := &WebFingerResponse{
+			Subject:    "ipns://$publicKeyHashValue",
```
Comment (bstasyszyn) [inline pkg/discovery/endpoint/restapi/operations.go:198]: What is this: $publicKeyHashValue?

Comment (fqutishat) [reply pkg/discovery/endpoint/restapi/operations.go:198]: This placeholder to replace with ipfs public key hash

Comment (fqutishat) [reply pkg/discovery/endpoint/restapi/operations.go]: I removed publicKeyHashValue placeholder

**Review by bstasyszyn — APPROVED** [review]

**Pull request merged**